include/wx/rawbmp.h

```diff
@@ -678,7 +696,7 @@ struct wxPixelDataOut<wxBitmap>
     // private: -- see comment in the beginning of the file
 
         // the bitmap we're associated with
-        wxBitmap m_bmp;
+        wxBitmap& m_bmp;
```

Comment:
Sorry, why? I don't see anything that requires changing this bitmap now.

Reply:
I thought the lack of the call to `AllocExclusive()` in `wxBitmap::GetRawData()` 
was a mistake (unless it's by design) because accessing raw bitmap data should do a **COW** to break data sharing which is not the case currently. So if we (you?) decide to change `GetRawData()` implementation for each port to follow **wxDFB**:
https://github.com/wxWidgets/wxWidgets/blob/eb8a57760e61e72959593200d4df8d8b36dfb87a/src/dfb/bitmap.cpp#L506
then this change is necessary. If not, then I can revert this if you like ?